landingcorp/index.html

```diff
@@ -1,1634 +1,1588 @@
 <!DOCTYPE html>
 <html class="no-js" lang="">
```

Comment:
Seem like this entire file was largely rewritten by your editor. Please run `npm prettier --write landingcorp` before committing any source to avoid this issue :wink: